Jenkinsfile

```diff
@@ -872,9 +872,9 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
                                 def prefix = distPrefix()
                                 // If prefix is not empty then trimming trailing "-" from path as distPrefix() returns "PR-${CHANGE_ID}-".
                                 if (prefix){
-                                    prefix = prefix.substring(0, prefix.length() - 1)
+                                    prefix = prefix.substring(0, prefix.length() - 1) + '/'
                                 }
```

Comment:
Is there a stylistic preference for this cosmetic change? If you want to be OS-agnostic you can use `File.separator` (works in Java, should work in Groovy as well) but given we know what these slave nodes are running it's bit of a moot point...

Reply:
This fix is to address https://jira.suse.com/browse/CAP-405.

I guess there are lot of places where `/` is hard-coded, so unless we are planning to remove all references I would keep it like that.

Reply:
I don't think there are even any OSes around anymore that don't support `/` as the path separator. So I don't think it makes any sense to try to generalize things. Next thing you have to worry about is drive letters and VMS volume prefixes...

But in this case the string is to become part of a URL, in which case `/` is the only valid path separator; the local OS doesn't even come into play. So using a literal `/` is arguably *more* correct. 😄

Reply:
My point was why make this change at all - something I still don't understand. Anyway, it's a minor nit...

Reply:
@satadruroy As I mentioned, this change has to do with resolving a bug in s3 log uploads, e.g: 

```
s3://cap-jenkins-logs/nightly//jenkins-scf-develop-1307/
```

Notice `//` in s3 path, we need to append it only if `prefix` is non NULL.